sql/7_0_1-to-7_0_2_upgrade.sql

```diff
@@ -577,4 +332,276 @@ ALTER TABLE `x12_partners` ADD COLUMN `x12_claim_status_endpoint` tinytext;
 
 #IfMissingColumn x12_partners x12_attachment_endpoint
 ALTER TABLE `x12_partners` ADD COLUMN `x12_attachment_endpoint` tinytext;
-#EndIf
\ No newline at end of file
+#EndIf
+
+#IfMissingColumn patient_data nationality_country
+ALTER TABLE `patient_data` ADD `nationality_country` TINYTEXT;
+#EndIf
+
+#IfRow2D list_options list_id lists option_id Nationality_and_Country
+DELETE FROM `list_options` WHERE  `list_id` = 'lists' AND `option_id` = 'Nationality_and_Country';
+#EndIf
+
+#IfRow list_options list_id Nationality_and_Country
+DELETE FROM `list_options` WHERE  `list_id` LIKE 'Nationality_and_Country';
+#EndIf
+
+#IfRow2D layout_options form_id DEM field_id nationality_country
```

Comment:
actually to be safe here (only ensure it runs once and doesn't overwrite future customizations), would do:
```
#IfRow3D layout_options form_id DEM field_id nationality_country list_id Nationality_and_Country
```